cheri-architecture.tex

```diff
@@ -56,6 +54,8 @@
 }
 
 %% CL tech-report format provides its own cover page
+\ifdefined\trformat
+\else
```

Comment:
There isn't a \ifundefined in TeX.  There is a \@ifundefined in LaTeX but the first time you use it defines its argument, hence using this construct with an empty \if body.